features/bsm/service/impl/src/main/java/org/opennms/netmgt/bsm/service/internal/BusinessServiceManagerImpl.java

```diff
@@ -130,38 +131,22 @@ public boolean removeIpInterface(Long serviceId, Integer ipServiceId) {
     }
 
     @Override
-    public List<IpServiceDTO> getAllIpServiceDTO() {
-        return transformAll(monitoredServiceDao.findAll());
+    public OnmsSeverity getOperationalStatusForBusinessService(Long serviceId) {
+        final BusinessService service = getBusinessService(serviceId);
+        final OnmsSeverity severity = businessServiceStateMachine.getOperationalStatus(service);
+        return severity != null ? severity : OnmsSeverity.INDETERMINATE;
```

Comment:
I guess, this should return OnmsSeverity.NORMAL instead of INDETERMINATE, otherwise the IT will fail (see below).